### DISCUSSION TITLE
No console logs when testing

### DISCUSSION THREAD

**Discussion by DevWedeloper** [analogjs/analog#1309] [General]
### Please provide the environment you discovered this bug in.

Windows 10

### Which area/package is the issue in?

vitest-angular

### Description

console.logs aren't appearing on the terminal when running tests

### Other information

Similar issue [here](https://github.com/vitest-dev/vitest/issues/5887)

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by gultyayev** [comment]
One way is to call `vitest --disableConsoleIntercept`.

Another, is to tinker the config:

```
export default defineConfig(({ mode }) => {
  return {
    plugins: [angular(), nxViteTsPaths()],
    test: {
      globals: true,
      environment: 'jsdom',
      setupFiles: ['src/test-setup.ts'],
      include: ['**/*.spec.ts'],
      reporters: ['default'],
      disableConsoleIntercept: true, // <-- here
    },
    define: {
      'import.meta.vitest': mode !== 'production',
    },
  };
});
```

Not sure why it's not present in the docs, but the IDE discovers it perfectly fine, and it also works 
<img width="752" alt="image" src="https://github.com/analogjs/analog/assets/22374911/ef66ebe2-c616-41c7-9744-b4efc468e7fa">

**Comment by DevWedeloper** [comment]
I have tried that on a fresh analog app and it's not working.

**Comment by gultyayev** [comment]
Could you provide a repro

**Comment by DevWedeloper** [comment]
> Could you provide a repro

Can you make a fresh analog app and test on your end? Just to make sure I'm not having an isolated case

**Comment by gultyayev** [comment]
Could be related https://github.com/vitest-dev/vitest/pull/5678
